content/partners/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Pulumi Partners
+layout: partners
+
+meta_title: Pulumi Partners - Accelerate Your Cloud Journey
+meta_desc: Explore Pulumi Professional Services for expert guidance on cloud infrastructure, automation, and best practices to streamline your cloud journey.
+
+overview:
+    title: Accelerate Your Cloud Journey
+    description: |
+        Our mission is to empower our partners with the tools and resources they need to build robust, cloud-native infrastructure solutions. Comprehensive guides, detailed documentation, an engaged community, and expert support make partnering with us a seamless and productive experience. With our infrastructure-as-code platform and extensive API suite, you can create customized, scalable, and secure infrastructure solutions for your clients.
+    form:
+        title: Contact our Partner Team
+        description: Ready to enhance your customers' cloud infrastructure with powerful automation and infrastructure-as-code capabilities? Drop our partner team a note to get started with Pulumi.
+        form_id: d8eaf931-a94c-43cc-92d3-57557c6babf2
+        campaign_id: 701PQ00000DrAWQYA3
+
+types:
+    title: Partnership Types
+    items:
+        - title: Cloud Partners
+          icon:
+            name: cloud
+            color: yellow
+          description: World’s leading hyperscalers that provide world class service to our mutual customers
+        - title: Technology Partners
+          icon:
+            name: nodes
+            color: salmon
+          description: Independent vendors integrate and test their products with Pulumi, enabling comprehensive, end-to-end cloud infrastructure solutions
+        - title: System Integrators
+          icon:
+            name: shield
+            color: purple
+          description: Trusted and experienced advisors that provide customized solutions to clients implementing modern Infrastructure as Code
+
```

Comment:
Below the list of partners, include a link for "See our ecosystem partners" and link to the Pulumi Registry